app/Livewire/Project/Service/ImportBackup.php

```
@@ -0,0 +1,79 @@
+        $environment = $project->environments()->whereUuid($this->parameters['environment_uuid'])->firstOrFail();
+        $this->service = $environment->services()->whereUuid($this->parameters['service_uuid'])->firstOrFail();
+        $this->authorize('update', $this->service);
+
+        $this->databases = $this->service->databases
+            ->filter(fn (ServiceDatabase $database): bool => $this->supportsImport($database))
+            ->values();
+
+        $databaseUuid = request()->route('stack_service_uuid');
+        if ($databaseUuid) {
+            $this->selectedDatabase = $this->databases->firstWhere('uuid', $databaseUuid);
+            abort_unless($this->selectedDatabase, 404);
```

PHPStan (2.2.8)

[warning] Parameter `#1` $boolean of function abort_unless expects bool, App\Models\ServiceDatabase|null given.


(argument.type)